### PR TITLE
mongoose-paginate-v2: support read opts, mongoose: tiny typo fix

### PR DESCRIPTION
**Pull request by chazepps**
* mongoose-paginate-v2: support read opts
* mongoose: tiny typo fix

Please fill in this template.

- [v] Use a meaningful title for the pull request. Include the name of the package modified.
- [v] Test the change in your own code. (Compile and run.)
- [v] Add or edit tests to reflect the change. (Run with `npm test`.)
- [v] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [v] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [v] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [v] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/ChazEpps/mongoose-paginate-v2/commit/9bd165499062be113b2e9359a84e50bb2a7ffba5
- [v] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by oktapodia — CHANGES_REQUESTED** [review]
Thank you for your PR, looks good but [this](https://github.com/aravindnc/mongoose-paginate-v2/pull/56) needs to be merged first

**Comment by oktapodia** [comment]
Related to https://github.com/aravindnc/mongoose-paginate-v2/pull/56

**Comment by ChazEpps** [comment]
@okamiconcept  Thank you for your Review 😄 
Yes, I'm waiting for merge. I'll let you know when the merge is complete.

**Comment by ChazEpps** [comment]
https://github.com/aravindnc/mongoose-paginate-v2/pull/56

@oktapodia p/r merged!

**Review by oktapodia — APPROVED** [review]

**Comment by oktapodia** [comment]
Thank you for your PR

**Comment by ChazEpps** [comment]
@horiuchi Thank you for your review 😸

**Comment by orta** [comment]
Looks good 👍

**Pull request merged**
